monitor/api/alerts.py

```diff
@@ -40,21 +42,50 @@ def _query_alert_type(self, run_operation_args: dict, alert_factory_func: Callab
         if raw_alerts:
             alert_dicts = json.loads(raw_alerts[0])
             for alert_dict in alert_dicts:
+                notmalaized_alert = self._normalize_alert(alert=alert_dict)
```

Comment:
if normalize_alert throws an exception, we don't want to crash the entire run. IMO - we should keep it in the try-except and use the MalformedAlert on the original alert as a last resort. WDYT?